Bluepill-runner/Bluepill-runner/BPRunner.m

```diff
@@ -88,7 +88,7 @@ - (NSTask *)newTaskWithBundle:(BPXCTestFile *)bundle
     assert(cfg);
     cfg.appBundlePath = bundle.testHostPath;
     cfg.testBundlePath = bundle.testBundlePath;
-    if (bundle.UITargetAppPath) cfg.testRunnerAppPath = bundle.UITargetAppPath;
+    cfg.testRunnerAppPath = bundle.UITargetAppPath ?: nil;
```

Comment:
actually, `bundle.UITargetAppPath` will either be a path or nil, yes? Maybe a straight assignment would be okay?